### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by srikanthmalla** [AI-liu/Complex-YOLO#6]
Hi @AI-liu ,
You are not doing validation output to check if it overfits to the data. Using same hyperparameters from the repo, I trained it for 200 epochs on KITTI. It overfits to the train data. Your evaluation when plotting on first 100. It only gives outputs for train dataset not for validatation dataset. Did you use same code and hyperparams for your results in the repo?

**Comment by abhigoku10** [comment]
@srikanthmalla  hi did u added validation block into the training and save the best loss ??
